docs/how_to_add_packages.md

```diff
@@ -196,16 +200,19 @@ The [recipes](https://github.com/conan-io/conan-center-index/tree/master/recipes
 
 ### Header Only
 
-If you are looking for header-only projects, you can take a look on [rapidjson](https://github.com/conan-io/conan-center-index/blob/master/recipes/rapidjson/all/conanfile.py), [rapidxml](https://github.com/conan-io/conan-center-index/blob/master/recipes/rapidxml/all/conanfile.py), and [nuklear](https://github.com/conan-io/conan-center-index/blob/master/recipes/nuklear/all/conanfile.py). Also, Conan Docs has a section about [how to package header-only libraries](https://docs.conan.io/en/latest/howtos/header_only.html).
+If you are looking for header-only projects, you can take a look on [header-only template](package_templates/header_only).
+Also, Conan Docs has a section about [how to package header-only libraries](https://docs.conan.io/en/latest/howtos/header_only.html).
 
 ### CMake
 
-For C/C++ projects which use CMake for building, you can take a look on [szip](https://github.com/conan-io/conan-center-index/blob/master/recipes/szip/all/conanfile.py) and [recastnavigation](https://github.com/conan-io/conan-center-index/blob/master/recipes/recastnavigation/all/conanfile.py).
+For C/C++ projects which use CMake for building, you can take a look on [cmake package template](package_templates/cmake_package).
 
 #### Components
 
 Another common use case for CMake based projects, both header only and compiled, is _modeling components_ to match the `find_package` and export the correct targets from Conan's generators. A basic examples of this is [cpu_features](https://github.com/conan-io/conan-center-index/blob/master/recipes/cpu_features/all/conanfile.py), a moderate/intermediate example is [cpprestsdk](https://github.com/conan-io/conan-center-index/blob/master/recipes/cpprestsdk/all/conanfile.py), and a very complex example is [OpenCV](https://github.com/conan-io/conan-center-index/blob/master/recipes/opencv/4.x/conanfile.py).
 
+Note that Conan is migrating toward the version 2.0 and many repices are outdated when related to components feature. To understand who to migrate components read [properties migration](https://docs.conan.io/en/latest/migrating_to_2.0/properties.html#properties-migration).
```

Comment:
what exactly does it mean? `many repices are outdated when related to components feature`.
should I (as a reader or as a consumer) be worried about it? should I update something specific?

Reply:
Current "real" examples can be outdated, when copied. But indeed it should be updated for a better comprehension

Reply:
yes, I think it's too confusing for newcomers.
if something should be updated when copied, then it should be clearly specified, like `replace "X" with "Y"`.
or even better, it should use already updated recipe as an example, or we should just update `cpu_features`/`cpprestsdk`/`opencv`.
it's a bit counter-productive to recommend something that is already outdated for users.

Reply:
Yes, just removed that paragraph, as those recipes support both V1 and V2, which means, newcomers can read to have some insight without further explanation now.